daffodil-test/src/test/scala/org/apache/daffodil/section12/lengthKind/TestLengthKindPrefixed.scala

```diff
@@ -150,4 +150,13 @@ class TestLengthKindPrefixed {
   @Test def test_pl_lengthunits_err() = { runner.runOneTest("pl_lengthunits_err") }
   @Test def test_pl_nest_err() = { runner.runOneTest("pl_nest_err") }
   @Test def test_pl_decimal_err() = { runner.runOneTest("pl_decimal_err") }
+
+  // DAFFODIL-2657
+  // @Test
+  def test_pl_implicit_1() = { runner.runOneTest("pl_implicit_1")}
+
+  // DAFFODIL-2656
+  // @Test
+  def test_pl_contentLength_1() = { runner.runOneTest("pl_contentLength_1")}
```

Comment:
Good idea to comment out only the @Test so the rest of the line still can be checked for compilation errors.